src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -2823,7 +2823,7 @@ internal Exception BindRunspace(Runspace initializedRunspace, PSTraceSource runs
             // If a user has any module with the same name as that of the core module( or nested module inside the core module)
             // in his module path, then that will get loaded instead of the actual nested module (from the GAC - in our case)
```

Comment:
I think we should keep them separate commits, especially the one that rename the method names.